calculatorUICOOP/Views/MainPage.xaml

```diff
@@ -52,7 +52,7 @@
       
         <Button Command="{Binding DecimalInputCommand}" CommandParameter="." Style="{StaticResource numButton}" Text="." Grid.Row="5" Grid.Column="0"></Button>
         <Button Command="{Binding NumberInputCommand}" CommandParameter="0" Style="{StaticResource numButton}" Text="0" Grid.Row="5" Grid.Column="1"></Button>
-        <Button Command="{Binding EqualsInputCommand}" Style="{StaticResource numButton}" Text="=" Grid.Row="5" Grid.Column="2"></Button>
+        <Button Command="{Binding EqualsInputCommands}" Style="{StaticResource numButton}" Text="=" Grid.Row="5" Grid.Column="2"></Button>
```

Comment:
Not sure of the point of this change.

Reply:
Well the change in itself is good, because the actual command in the VM has that name but what should be done instead is to fix the name in the VM as it doesnt have much sense. EqualsInputCommands implies that theres some collection of Commands.